File: Examples/Ruby/Document/post_slides_split.rb
require 'aspose_slides_cloud'

class Document

  include AsposeSlidesCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("da77c9f5da014d91faf2961ecec2de66", "B01A15E5-1B83-4B9A-8EB3-0F2BFA6AC766")
    @slides_api = SlidesApi.new
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new 
    response = @storage_api.put_create(file_name, File.open("../data/" << file_name,"r") { |io| io.read } )
  end

  def post_slides_split
  	file_name = "sample-input.pptx"
  	upload_file(file_name)
  
  	response = @slides_api.post_slides_split(file_name, {to: 3, from: 2, format: "png"})
  end

end

document = Document.new()
puts document.post_slides_split

File: Examples/PHP/Document/PostSlidesSplit.php
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Slides\SlidesApi;
use Aspose\Slides\AsposeApp;

class Document {

	public $slidesApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->slidesApi = new SlidesApi();
	}

	public function postSlidesSplit() {
		$fileName = "sample-input.pptx";
		Utils::uploadFile($fileName);
		
		$result = $this->slidesApi->PostSlidesSplit($fileName, $width = "", $height = "", $to = "3", $from = "2", $destFolder = "", $format = "PNG", $storage = "", $folder = "");
		print_r($result);
	}
}

$document = new Document();
$document->postSlidesSplit();

?>